src/sage/env.py

```diff
@@ -515,3 +512,28 @@ def uname_specific(name, value, alternative):
     aliases["OPENMP_CXXFLAGS"] = OPENMP_CXXFLAGS.split()
 
     return aliases
+
+def sage_data_paths(data_name: str) -> set[str]:
+    r"""
+    Search paths for database ``data_name``.
```

Comment:
This can also be used in the future to search for other data files (eg. three.min.js), doesn't seem specific to databases.

Reply:
Right, this was a result of simply copy&pasting. Fixed now.